robot/control/Inc/modules/LEDModule.hpp

```diff
@@ -11,50 +11,89 @@
 #include "MicroPackets.hpp"
 #include "drivers/MCP23017.hpp"
 
+/**
+ * Module interfacing with debugging LEDS based on the statuses of other electronics
+ */
 class LEDModule : public GenericModule {
 public:
-    // How many times per second this module should run
-    static constexpr float kFrequency = 10.0f; // Hz
+    /**
+     * Number of times per second (frequency) that LEDModule should run (Hz)
+     */
+    static constexpr float kFrequency = 10.0f;
+
+    /**
+     * Number of seconds elapsed (period) between LEDModule runs (milliseconds)
+     */
     static constexpr std::chrono::milliseconds kPeriod{static_cast<int>(1000 / kFrequency)};
+
+    /**
+     * Priority used by RTOS
+     *
+     * Uses the default priority of 1.
+     */
     static constexpr int kPriority = 1;
 
+    /**
+     * Constructor for LEDModule
+     *
+     * @param ioExpander Interfaces with MCP23017
+     * @param dotStarSPI SPI bus for dotStar LEDS
+     * @param batteryVoltage Packet of data containing data on battery voltage and critical status
+     * @param fpgaStatus Packet of data containing whether motors or FPGA have errors
+     * @param kickerInfo Packet of data containing kicker status
+     * @param radioError Packet of data containing whether radio has an error
+     */
     LEDModule(LockedStruct<MCP23017>& ioExpander,
               LockedStruct<SPI>& dotStarSPI,
               LockedStruct<BatteryVoltage>& batteryVoltage,
               LockedStruct<FPGAStatus>& fpgaStatus,
               LockedStruct<KickerInfo>& kickerInfo,
               LockedStruct<RadioError>& radioError);
 
+    /**
+     * Code which initializes module
+     */
     void start() override;
 
+    /**
+     * Code to run when called by RTOS
+     *
+     * Checks for Radio, FPGA, and Kicker errors, and activates dotStar LEDs accordingly
+     */
     void entry() override;
 
-    // Specific LED pattern for fpga initialization
+    /**
+     * Set specific LED pattern for fpga initialization
+     */
     void fpgaInitialized();
```

Comment:
For these `*Initialized()` functions, fpga turns on LED1, radio turns on LED2, kicker turns on LED3, fully turns on LED4, resulting in a loading bar if you watch the startup sequence